Libs/MRML/DisplayableManager/vtkThreeDViewInteractorStyle.cxx

```
@@ -792,6 +783,56 @@ void vtkThreeDViewInteractorStyle::SetModelDisplayableManager(
 }
 
 //----------------------------------------------------------------------------
+void vtkThreeDViewInteractorStyle::RotateToPosterior(bool inverse)
+{
+  if (!this->CameraNode)
+    {
+    return;
+    }
+
+  this->CameraNode->RotateTo(
+    inverse ? vtkMRMLCameraNode::Posterior : vtkMRMLCameraNode::Anterior);
+}
+
+//----------------------------------------------------------------------------
+void vtkThreeDViewInteractorStyle::RotateToRight(bool inverse)
+{
+  if (!this->CameraNode)
+    {
+    return;
+    }
+
+  this->CameraNode->RotateTo(
+    inverse ? vtkMRMLCameraNode::Right : vtkMRMLCameraNode::Left);
+}
+
+//----------------------------------------------------------------------------
+void vtkThreeDViewInteractorStyle::RotateToInferior(bool inverse)
+{
+  if (!this->CameraNode)
+    {
+    return;
+    }
+
+  this->CameraNode->RotateTo(
+    inverse ? vtkMRMLCameraNode::Inferior : vtkMRMLCameraNode::Superior);
+}
+
+//----------------------------------------------------------------------------
+void vtkThreeDViewInteractorStyle::
+ResetCamera(bool resetRotation, bool resetTranslation, bool resetDistance)
+{
+  if (!this->CameraNode)
+    {
+    return;
```

log warning instead of just silently fail